its/src/test/java/com/sonar/it/scanner/msbuild/TestUtils.java

```diff
@@ -245,6 +246,20 @@ static void dumpComponentList(Orchestrator orchestrator)
     }
   }
 
+  public static List<Issue> issuesForComponent(Orchestrator orchestrator, String componentKey) {
```

Comment:
I copied this method from the fix MichaelG did for the Java ITs: https://github.com/SonarSource/sonar-java/pull/2486/files#diff-65fbd0968dc57fefb6c158c81eb2088aR73